src/sparql/icd10cm-relevant-diseases.sparql

```diff
@@ -33,5 +33,5 @@ WHERE {
       }
     }
   }
-  FILTER(isIRI(?term))
+  FILTER(isIRI(?term) && (regex(str(?term), "http://purl.bioontology.org/ontology/ICD10CM/")))
```

Comment:
this is what I did in my other PR, so this should not be a conflict when I merge.